doc/tags

```diff
@@ -0,0 +1,26 @@
+buffer's	lspconfig.txt	/*buffer's*
```

Comment:
typo: `buffers`?

Reply:
That is autogenerated by `:helptags` coming from this line https://github.com/neovim/nvim-lspconfig/blob/1c9c2120290d2fd6ea2cb55f954b5d108df0b628/doc/lspconfig.txt#L484 I'm guessing mj used that syntax to have some accent and it generates that on the tags.

Reply:
Gotcha. (I'd have written that as `*buffer*'s`, but to each their own ;))

Reply:
Oh that tag shouldn't be there... Let's just change lspconfig.txt